cli/cmd/outputs.go

```diff
@@ -64,3 +66,46 @@ func (c *cliState) FormatJSONString(s string) (string, error) {
 	}
 	return string(pretty), nil
 }
+
+// Used to clean CSV inputs prior to rendering
+func csvCleanData(input []string) []string {
+	var data []string
+	for _, h := range input {
+		data = append(data, strings.Replace(h, "\n", "", -1))
+	}
+	return data
+}
+
+// Used to produce CSV output
+func renderAsCSV(headers []string, data [][]string) (error, string) {
+	csvOut := &strings.Builder{}
+	csv := csv.NewWriter(csvOut)
+
+	if len(headers) > 0 {
+		if err := csv.Write(csvCleanData(headers)); err != nil {
+			return errors.Wrap(err, "Failed to build csv output"), ""
+		}
+	}
+
+	for _, record := range data {
+		if err := csv.Write(csvCleanData(record)); err != nil {
+			return errors.Wrap(err, "Failed to build csv output"), ""
+		}
+	}
+
+	// Write any buffered data to the underlying writer (standard output).
+	csv.Flush()
+	return nil, csvOut.String()
+}
+
+// OutputCSV will print out the provided headers/data in CSV format if the cli state is
+// configured to output CSV
+func (c *cliState) OutputCSV(headers []string, data [][]string) error {
+	err, csv := renderAsCSV(headers, data)
+	if err != nil {
+		return err
+	}
+
+	fmt.Print(csv)
```

Comment:
We prefer to control the file descriptor where we will print out, lets follow the above standard:
```suggestion
	fmt.Fprint(os.Stdout, csv)
```